about.html

```diff
@@ -0,0 +1,121 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./assets/css/style.css">
+    <script src="https://unpkg.com/ionicons@4.5.10-0/dist/ionicons.js"></script>
+    <script type="text/javascript" src="http://code.jquery.com/jquery-latest.js"></script>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <title>Web Study</title>
+</head>
+<body>
+    <!--======================== Header =========================!-->
+<section class="header">
+  <div class="top-header">
+    <ul>
+     <li><ion-icon name="logo-facebook" class="top-icon"></ion-icon></li>
+     <li><ion-icon name="logo-twitter" class="top-icon"></ion-icon></li>
+     <li><a href="#">English</a></li>
+     <li><a href="#">My Page</a></li>
+     <li><a href="#">Logout</a></li>
+    </ul>
+   </div>
+   <div class="header-desk">
+    <a href="index.html" class="logo"><img src="./assets/images/logo.png" alt="logo"/></a>
+   <nav class="navbar">
+      <ul class="nav-links">
+      <li class="nav-item "><a href="index.html">Home</a> </li>
+      <li class="nav-item active"><a href="about.html">About</a></li>
+      <li class="nav-item"><a href="#speaker">Speakers</a></li>
+      <li class="nav-item"><a href="#partnersid">Partners</a></li>
+      </ul>
+    </nav> 
+    </div>
+    <div class="hamburger-icon" id="menuicon">
+      <div class="bar1"></div>
+      <div class="bar2"></div>
+      <div class="bar3"></div>
+    </div>
+    <div id="menu-">
+        <div id="closed">
+          <span class="close">&times;</span>
+        </div>
+        <div id="menu-list">
+        <ul id="mlist">
+          <li><a href="index.html">Home</a></li><hr>
+          <li><a href="about.html">About</a></li><hr>
+          <li><a href="#speaker">Speakers</a></li> <hr>
+          <li><a href="#partnersid">Partners</a></li> <hr>
+        </ul>
+      </div>
```

Comment:
- [ ] Please cover the whole page with navigation menu on mobile screens and be sure to disable horizontal scrolling when menu appears. That would be more professional
![image](https://user-images.githubusercontent.com/91486791/203619613-8448a398-0224-4247-8047-7ace715bb24f.png)